src/internally.rs

```diff
@@ -111,7 +114,14 @@ impl<'de, T: ?Sized> Visitor<'de> for TaggedVisitor<T> {
 
         let deserialize_fn = match deserialize_fn {
             Some(deserialize_fn) => deserialize_fn,
-            None => return Err(de::Error::missing_field(self.tag)),
+            None => match self.default_variant {
+                Some(variant) => if let Some(Some(deserialize_fn)) = self.registry.map.get(variant) {
+                    *deserialize_fn
+                } else {
+                    return Err(de::Error::unknown_variant(variant, &self.registry.names));
+                },
```

Comment:
Again using `map_lookup`:

```suggestion
                Some(variant) => map_lookup.visit_str(variant)?,
```